src/server/user_handlers.py

```diff
@@ -132,7 +132,9 @@ def create_new_repository(
         salt=secrets.token_bytes(32),
     )
 
-    db.create_new_repository(repo_id, repository.owner.id, digest)
+    db.create_new_repository(
+        repo_id, repository.owner.id, digest, repository.full_name, "BUCKET PLACEHOLDER"
+    )
```

Comment:
i think we'll need to create a bucket when the vault is initialized, or we could create it lazily for when the first env is created. what is your thought?